crates/net/downloaders/src/headers/metrics.rs

```diff
@@ -0,0 +1,40 @@
+use metrics::Counter;
+use reth_interfaces::p2p::error::DownloadError;
+use reth_metrics_derive::Metrics;
+
+/// The header downloader metrics.
+#[derive(Metrics)]
+#[metrics(scope = "downloaders_headers")]
+pub struct HeaderDownloaderMetrics {
+    /// The number of headers that were successfully returned by the downloader.
+    total_returned: Counter,
```

Comment:
Not sure about the naming and comment here. I understand what it is saying, but I think w/o my context it wouldn't make a lot of sense

Maybe `total_flushed`? The comment should at least specify that this is the number of headers sent to the stage.